GSA/code-gov-front-end
# Fix required data on Metadata Schema 2.0.0 (version)

**Summary**

This PR fixes the following **bugs/features**

* [ ] #222  

**Test plan (required)**
![fix-metadata](https://user-images.githubusercontent.com/5417662/59417940-0e05a900-8dc0-11e9-98f8-714a9ade21da.gif)

1. Go to [http://localhost:8080/about/compliance/inventory-code](http://localhost:8080/about/compliance/inventory-code)
2. Click on the **Hide optional fields checkbox**.
3. Notice how the list **`doesn't`** removes **method** and **ifOther** from the list.


**Closing issues**

Closes #222
